modules/phase_field/test/tests/anisotropic_mobility/ad_diffusion.i

```diff
@@ -0,0 +1,63 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 15
+  ny = 15
+  xmax = 15.0
+  ymax = 15.0
+  elem_type = QUAD4
```

Comment:
unnecessary elem_type?

Reply:
yeah, here and in hundreds of inputs. I'll remove this one